Userland/Libraries/LibArchive/Zip.h

```diff
@@ -225,7 +225,7 @@ struct [[gnu::packed]] LocalFileHeader {
 };
 
 struct ZipMember {
-    DeprecatedString name;
+    String name = String();
```

Comment:
This explicit initialization looks a bit suspicious. Do we not always have something to put there as soon as we actually create a new ZipMember?